### PR TITLE
Update protein subunit

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Hey @RLordan, here are some edits to the protein subunit section. I am going through again (😬) to try to trim and move text from the paper as a whole, so there will probably be more PRs this week with those edits! I had a grant due 2/28 but I'm back to aggressively trying to wrap this manuscript up now!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/22.vaccines.md [hunk 1]
```diff
@@ -343,39 +333,22 @@ These data are retrieved from Our World in Data <!-To Do: Cite--> and plotted us
...
-<!--To Do: add info about VLP and phase 1/2 trials:
-https://www.raps.org/news-and-articles/news-articles/2020/3/covid-19-vaccine-tracker
-https://pubmed.ncbi.nlm.nih.gov/20923267/
-https://jnanobiotechnology.biomedcentral.com/articles/10.1186/s12951-021-00806-7
--->
-
-##### Recombinant Protein Vaccine
-
-<!--To Do: add, see https://www.raps.org/news-and-articles/news-articles/2020/3/covid-19-vaccine-tracker-->
+Novavax is now preparing to apply for an EUA to the FDA [@url:https://ir.novavax.com/Novavax-Statement-on-UK-and-Mexico-Phase-3-Clinical-Trial-Participants-Considered-Fully-Vaccinated-in-the-US].<!--Still true?-->
```
Comment (agitter) [inline content/22.vaccines.md:339]: It looks like they submitted the EUA in January: https://ir.novavax.com/2022-01-31-Novavax-Submits-Request-to-the-U-S-FDA-for-Emergency-Use-Authorization-of-COVID-19-Vaccine

The Wall Street Journal reported today that they expect approval in the coming weeks (original article paywalled): https://fortune.com/2022/03/07/fourth-covid-vaccine-novavax-anti-vax-protein-mrna/

Comment (rando2) [reply content/22.vaccines.md:339]:
```suggestion
Novavax applied for an EUA from the FDA in early 2022 [@url:https://ir.novavax.com/2022-01-31-Novavax-Submits-Request-to-the-U-S-FDA-for-Emergency-Use-Authorization-of-COVID-19-Vaccine].
```
Thank you @agitter! New motivation to try to get this out before anything else changes 😆

File: content/22.vaccines.md [hunk 2]
```diff
@@ -120,20 +120,21 @@ As of {{owid_most_recent_date}}, {{owid_vaccine_types}} types of vaccines are be
...
+
 #### Whole-Virus Vaccines
 
 Whole-virus vaccines have the longest history among vaccine development approaches. 
 The first vaccination strategy in human history is widely considered to be the practice of variolation, which makes the history of vaccine development almost 500 years long [@doi:10.1016/j.jinf.2005.07.021; @doi:10.1007/978-1-4419-1339-5_2].
 Famously employed as a strategy to improve survival of smallpox by, for example, exposing a healthy individual to pus from smallpox pustules [@doi:10.1377/hlthaff.24.3.611; @doi:10.1016/j.jinf.2005.07.021; @doi:10.1007/978-1-4419-1339-5_2], variolation provides a mechanism for infecting a healthy individual with a mild case of a disease.
 This strategy aims to confer adaptive immunity, but it also carries a number of risks for the vaccine recipient [@doi:10.1016/j.virol.2015.03.032].
-The approach was (debatably) the first example of a live-attenuated virus being used to induce immunity [@doi:10.1016/j.virol.2015.03.032; 10.1073/pnas.1400472111].
+The approach was (debatably) the first example of a live-attenuated virus being used to induce immunity [@doi:10.1016/j.virol.2015.03.032; @doi:10.1073/pnas.1400472111].
```
Comment (RLordan) [inline content/22.vaccines.md:131]:
```suggestion
The approach was (debatably) the first example of a live-attenuated virus used to induce immunity [@doi:10.1016/j.virol.2015.03.032; @doi:10.1073/pnas.1400472111].
```

**Review by RLordan — APPROVED** [review]
Hey @rando2 I think this is good to go.

**Pull request merged**